third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -391,8 +392,11 @@ void dds_device::impl::write_control_message( topics::flexible_msg && msg, nlohm
         {
             throw std::runtime_error( "timeout waiting for reply #" + std::to_string( this_sequence_number ) );
         }
+        LOG_DEBUG( "got reply: " << actual_reply );
         *reply = std::move( actual_reply );
+        LOG_DEBUG( "move OK" );
```

Comment:
I don't think we should leave this debug message.

Reply:
Right, leftover, will remove